torch/_inductor/codegen/cpp.py

```diff
@@ -1308,11 +1315,21 @@ def truncdiv(a, b):
 
     @staticmethod
     def minimum(a, b):
-        return f"at::vec::minimum({a}, {b})"
+        if a.dtype == torch.bool:
+            assert b.dtype == torch.bool
+            a_cast, b_cast = unify_mask_base_type(V.kernel.compute, (a, b))
+            return f"decltype({a_cast})::minimum({a_cast}, {b_cast})"
```

Comment:
NIT: `min(a, b)` on boolean is just `a & b`.  Similarly `max(a, b)` is `a | b`.

Reply:
Thanks for the comment. Simplify to use `a & b` and `a | b`.